src/components/app-controllers/SingleAppController/SingleAppController.js

```diff
@@ -36,52 +40,52 @@ export default createReactClass({
   //only datasets the filter returns a truthy value for are available.
   //Filters out noisy multi-year monthly datasets.
   datasetFilter: function (datafile) {
-    return !(datafile.multi_year_mean == false && datafile.timescale == "monthly");
+    return !(datafile.multi_year_mean === false && datafile.timescale === 'monthly');
   },
 
   //Returns metadata for datasets with thethe selected variable + scenario, any model.
   //Passed as a prop for SingleDataController to generate model comparison graphs.
   getModelContextMetadata: function () {
     return _.where(this.state.meta,
-        {variable_id: this.state.variable_id, experiment: this.state.experiment});
+        { variable_id: this.state.variable_id, experiment: this.state.experiment });
```

Comment:
Sorry, several changes in this file are due to automatically complying with our standard ESLint conventions, and really have nothing to do with this PR's actual purpose. They can be removed if you find them too distracting.